manifests/params.pp

```diff
@@ -145,22 +151,40 @@
         }
       }
 
+    }
+    'Debian': {
       $service_name       = 'elasticsearch'
       $service_hasrestart = true
       $service_hasstatus  = true
       $service_pattern    = $service_name
-      $defaults_location  = '/etc/sysconfig'
-      $pid_dir            = '/var/run/elasticsearch'
+      $defaults_location  = '/etc/default'
+
+      if ($::operatingsystemmajrelease >= 8) {
```

Comment:
Forgot i enabled [this](https://github.com/puppet-community/puppet-lint-version_comparison-check) check which causes puppet-lint now to fail.
See https://github.com/puppet-community/puppet-lint-version_comparison-check#version-compared-as-number for details.